### DISCUSSION TITLE
Android Studio

### DISCUSSION THREAD

**Discussion by FabyoSK** [catppuccin/catppuccin#2150] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by nekowinston** [comment]
Please see https://github.com/catppuccin/jetbrains/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aandroid-studio

We'd appreciate the help of someone using Android Studio over there.
